app/mailers/user_mailer.rb

```diff
@@ -10,9 +10,19 @@ def reset_password(user, token)
          subject: I18n.t('user_mailer.reset_password.subject'))
   end
 
+  def invite(email, token)
+    @email = email
+    @link = invite_link(token)
+    mail(to: @email, subject: I18n.t('user_mailer.invite.subject'))
+  end
+
   private
 
   def token_link(token)
     URI.parse("#{Rails.application.config.client_url}/reset_password?token=#{token}").to_s
   end
+
+  def invite_link(token)
```

Comment:
For future - it is more better if backend get redirect url from frontend and then backend only add token param to this url